assignment/api/src/resources/assignment.properties

```diff
@@ -162,7 +162,7 @@ gen.mod           = Modified
 gen.modbyins      = Modified by instructor
 gen.new           = New
 gen.no            = No
-gen.noasscan      = Assignments cannot be submitted after the close date.
+gen.noasscan      = Assignments cannot be submitted after the until date.
```

Comment:
This is awkward phrasing!

Reply:
I have some wording suggestions for this coming....